R/io.R

```diff
@@ -26,15 +30,15 @@ check_package <- function(pkg, file = NULL) {
 #' qload("examples/wine", "quality")
 #' qload("examples/wine", "quality/red")
 #' qload("akarve/seattle_911", "responses")
-qload <- function(pkg, file, ...) {
+qload <- function(pkg, file, file_type = "feather", package_path = "~/quilt_packages/") {
```

Comment:
`file_type` is also probably also a good place to use an R option to set the default